openpype/hosts/flame/api/utility_scripts/openpype_flame_to_ftrack/openpype_flame_to_ftrack.py

```diff
@@ -3,6 +3,11 @@
 import os
 import sys
 
+try:
+    import six  # noqa
```

Comment:
Why this module would not be available?

Reply:
this file is copied to flame shared folder where it could be available for running flame session even not started from OpenPype. This was causing the issue, where the tools was offered to User but since it was not run from OpenPype with all dependencies, it had just simply not offer to user the interface. The only way to disable the offering was to check of any of the dependencies and if they are not available then the crashing of the hook will prevent from showing the menu item to user.

Reply:
So why you're catching `ImportError` and raising `ImportError`? Default message tells that can't import module and you're telling that can't import module because can't import module.